ui-tests/src/test/resources/features/integrations/mqtt.feature

```diff
@@ -1,7 +1,6 @@
 # @sustainer: mcada@redhat.com
 
 @ui
-@activemq
```

Comment:
shoudln't this stay here?

Reply:
I removed it because the test uses MQTT connector. I use @activemq tag only when a test uses AMQ connector (Red Hat AMQ connector now). e.g. @activemq tag is not in the ampq test either.
But it is true that those tests (mqtt, ampq) use AMQ broker.
So I can use @activemq tag for all tests where AMQ broker is present and @redhatamq for tests where the Red Hat AMQ connector is used.

What Do You Think?

Reply:
What about `@broker` for all the tests with broker and `@activemq` for the syndesis activemq connector?

Reply:
LGTM

Reply:
may be